tests/functional/disaster-recovery/regional-dr/test_failover_and_relocate_discovered_apps.py

```diff
@@ -28,28 +28,36 @@ class TestFailoverAndRelocateWithDiscoveredApps:
     """
 
     @pytest.mark.parametrize(
-        argnames=["primary_cluster_down", "pvc_interface"],
+        argnames=["primary_cluster_down", "pvc_interface", "kubeobject", "recipe"],
```

Comment:
Create a new test file recipe workloads

Reply:
Please correct me if i am wrong, why do we need a new test file for recipe when we all other DR OPS like failover/relocate and validations are similar. We can enhance the fixture in the near future instead of having a new test file https://github.com/red-hat-storage/ocs-ci/issues/11903

Reply:
if we want to have in single file, you need to add more values right now you reduced combination

Reply:
Add more more combinations for kubeobject=1, recipe=0